lib/LLVMIRCodeGen/LLVMIRGen.cpp

```diff
@@ -66,8 +66,8 @@ llvm::cl::opt<llvm::Reloc::Model> relocModel(
 /// Limitation of number of arguments for `emitDataParallelKernel`.
 constexpr static size_t kArgLimit = 64;
 
-/// Generate the LLVM MAttr list of attributes.
-static llvm::SmallVector<std::string, 0> getMachineAttributes() {
+/// Generate the LLVM machine attribute list for the host.
+static llvm::SmallVector<std::string, 0> getHostMachineAttributes() {
```

Comment:
I'd suggest submitting these LLVMIRGen.cpp changes (and the corresponding changes introducing floatABI options) as a small separate PR as it is a totally orthogonal issue.

Reply:
I reverted the changes and created a separate PR.